component/src/main/java/org/wso2/extension/siddhi/store/elasticsearch/ElasticsearchEventTable.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2018-2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Should be `2024, WSO2 LLC.`. Let's edit all the headers appropriately with the latest format

Reply:
Fixed with https://github.com/siddhi-io/siddhi-store-elasticsearch/pull/63/commits/1a954020321303d1c5af4a4ac630a2f280380dd3